db/migrate/20240916124923_add_ecf_id_to_remaining_tables.rb

```diff
@@ -0,0 +1,15 @@
+class AddEcfIdToRemainingTables < ActiveRecord::Migration[7.1]
+  def change
+    add_column :cohorts, :ecf_id, :uuid, null: true
```

Comment:
Do we need to also add `default: "gen_random_uuid()"` for new records going forward or will this be used only for the migration work?

Reply:
As we don't expose these models via the API we don't need to give them an `ecf_id` going forward; if they have the `ecf_id` set it indicates they were created as part of the migration (those with `nil` `ecf_id`s will indicate they were added after)

Reply:
Well yeah make sense..  those models I don't think we'll expose `ecf_id` in the future either.. tks @ethax-ross 👍